src/components/organisms/FileTreePane/FileTreePane.tsx

```diff
@@ -253,7 +253,7 @@ const FileTreePane = () => {
       setExpandedKeys(Object.keys(fileMap).filter(key => fileMap[key]?.children?.length));
       shouldExpandAllNodes.current = false;
     }
-  }, [fileMap]);
+  }, [resourceMap]);
```

Comment:
not both fileMap and resourceMap?

Reply:
I debugged as when we change the `fileMap` we also update the `resourceMap` so I thought there is no need to add fileMap into dependecies.